docs/api/python/api_summary.html

```diff
@@ -522,58 +522,7 @@ <h2><a class="toc-backref" href="#id6">Backend</a><a class="headerlink" href="#b
           
         </div>
       </div>
-      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
-        <div class="sphinxsidebarwrapper">
-            <p class="logo"><a href="index.html">
-              <img class="logo" src="static/ONNX_Runtime_icon.png" alt="Logo"/>
-            </a></p>
-<h1 class="logo"><a href="index.html">ONNX Runtime</a></h1>
 
-
-
-
-
-
-
-
-<h3>Navigation</h3>
-<ul class="current">
-<li class="toctree-l1"><a class="reference internal" href="tutorial.html">Tutorial</a></li>
-<li class="toctree-l1 current"><a class="current reference internal" href="#">API Summary</a></li>
-<li class="toctree-l1"><a class="reference internal" href="auto_examples/index.html">Gallery of examples</a></li>
```

Comment:
1) looks like we no longer have a link from anywhere to these auto_examples.
2) i could be wrong, but i think the examples are auto-generated. with the new structure, have the instructions for the team on how and where to generate and commit changes also been updated?

Reply:
I was playing with the layout by manually updating the html and removed them. Its confusing to have tutorials in multiple spots. The python docs are auto generated with Sphinx and I think we need to look at the python generation/format but I didnt get to it this round. For now they can continue with their normal process and when they update the docs next the links will be added back.